partner_firstname/res_user.py

```diff
@@ -1,43 +0,0 @@
-# -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Author: Nicolas Bessi. Copyright Camptocamp SA
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-from openerp.osv import orm
-from openerp.tools.translate import _
-
-
-class ResUsers(orm.Model):
-
-    _inherit = 'res.users'
-
-    def copy_data(self, cr, uid, _id, default=None, context=None):
```

Comment:
That code made that when copying a record it was stored with:

- lastname = "Previous lastname (copy)"
- firstname = "Previous firstname"

Removing that code makes it be copied as:

- lastname = "Previous lastname"
- firstname = "Previous firstname (copy)"

The default behavior seems good enough to me. Nothing of value is added with that code, as the user is expected to change those fields after copying, and I don't think its that important that the word "(copy)" appears in the last name instead of the first.

See [the test for that](https://github.com/OCA/partner-contact/pull/104/files#diff-7dde7b0bda5956cf9df1a6948c2fab9fR40). I removed it just for KISS.